### ISSUE TITLE
Peer dependencies need to be updated for eslint

### ISSUE THREAD

**Issue by PierBJX** [standard/eslint-config-standard#202]
**What version of this package are you using?**
`16.0.3`

**What operating system, Node.js, and npm version?**
- windows 10
- node 14.17.5
- npm 8.2.0

**What happened?**
With eslint v8.X.X, there is peer dependencies conflicts.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: myProject
npm ERR! Found: eslint@8.3.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"8.3.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^7.12.1" from eslint-config-standard@16.0.3
npm ERR! node_modules/eslint-config-standard
npm ERR!   dev eslint-config-standard@"16.0.3" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\212747277\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\212747277\AppData\Local\npm-cache\_logs\2021-12-23T15_12_55_367Z-debug-0.log
```

**What did you expect to happen?**
eslint-config-standard npm dependencies should be updated to fix this.

**Are you willing to submit a pull request to fix this bug?**
I can if you want

**Comment by aladdin-add** [comment]
will be fixed by https://github.com/standard/eslint-config-standard/pull/193 .

**Comment by tonix-tuft** [comment]
> will be fixed by #193 .

When? Thank you.
